01-main/packages/ulauncher

```diff
@@ -1,5 +1,10 @@
 DEFVER=1
```

Comment:
```suggestion
DEFVER=2
```

Reply:
EXTREPO.md: `DEFVER`: An integer greater than or equal to `1` that represents the version of the package definition. It must be incremented every time `ASC_KEY_URL`, `GPG_KEY_URL`, `APT_REPO_URL`, `APT_REPO_OPTIONS` or `PPA` are added, changed or removed, or the package installation method changes.